.github/ISSUE_TEMPLATE/submit-question.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Do we need submit-question.md? I suppose this is for discussion tab, which spark-rapids-tools doesn't have.